.github/workflows/build&deploy-dev.yml

```diff
@@ -0,0 +1,120 @@
+name: Shipping to AWS ECS from Source Code
+  
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  
+
+env:
+  APP_ENV: dev
+  AWS_REGION: us-west-2
+  ECR_REPOSITORY: billing-and-payment-service
```

Comment:
needs an ecr repo for dev env called dev-billing-and-payment-service?

Reply:
I will add "-dev" to the image tag, yes, we can use a different ecr repo as well. What's your choice?